docs/cloud-workload-security/backend_linux.md

```diff
@@ -3217,6 +3277,11 @@ Workload Protection events for Linux systems have the following JSON schema:
 | `change_time` | File change time |
 | `package_name` | System package name |
 | `package_version` | System package version |
+| `package_epoch` | System package epoch |
+| `package_release` | System package release |
+| `package_source_version` | System package source version |
+| `package_source_epoch` | System package source epoch |
+| `package_source_release` | System package source release |
```

Comment:
Can/Do we have functional tests for at least one of those new fields?

Reply:
Yes I am going to add more tests for those, in a follow up PR